src/Routers/routes/ConsultationRoutes.tsx

```diff
@@ -139,6 +140,6 @@ export default {
       />
     ),
   "/consultation/:consultationId": ({ consultationId }: any) => (
-    <ConsultationDetails consultationId={consultationId} tab={"updates"} />
+    <Redirect to={`/facility/id/patient/id/consultation/${consultationId}/`} />
```

Comment:
placing placeholder id as we don't know facility id and patient id

Reply:
Why? We could keep it as `<ConsultationDetails consultationId={consultationId} tab={"updates"} />` itself right? ConsultationDetails component would correct the URL for us right?

Reply:
This causes two redirects now:

- `/consultation/<valid_id>` goes to
- `/facility/id/patient/id/consultation/<valid_id>` goes to
- `/facility/<valid_id>/patient/<valid_id>/consultation/<valid_id>`

AppHistory stack would have unnecessary redirects + Back Button no longer works properly this way